session.go

```diff
@@ -1957,6 +1962,7 @@ func (t *traceWriter) Trace(traceId []byte) {
 type ObservedQuery struct {
 	Keyspace  string
 	Statement string
+	Values    []interface{}
```

Comment:
Could you please add documentation for the field?

For example:

```go
// Values holds a slice of bound values for the query.
// Do not modify the values here, they are shared with multiple goroutines.
Values     [][]interface{}
```

Reply:
@martin-sucha Documentation is added. Thanks